### DISCUSSION TITLE
ccf-validation-tools/Spleen/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#323] [General]
# ccf-validation-tools/Spleen/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Spleen/comments/

**Comment by anitacaron** [comment]
Spleen_v1.4_DRAFT: https://docs.google.com/spreadsheets/d/1KugrRJwk_IlvOpCpDffvk9YKFbSFd4PtJgAzxgrAmtM/edit?pli=1#gid=69626346

1. **FIXED** The term CL:0000037 has a different name/label in the source ontology in the following 1 row 36. The name/label in the ASCT+B table is hematopoetic stem cell and the one in the ontology is hematopoietic stem cell. For reference, the given name/label by SMEs is Hematopoetic Stem Cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.
1. **FIXED** The term CL:0002394 has a different name/label in the source ontology in the following 1 row 54. The name/label in the ASCT+B table is myeloid dendritic cell cDC1 and the one in the ontology is CD141-positive myeloid dendritic cell. For reference, the given name/label by SMEs is cDC1 myeloid dendritic cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.
1. **FIXED** The term UBERON:0009967 has a different name/label in the source ontology in the following 1 row 30. The name/label in the ASCT+B table is splenic sinusoid and the one in the ontology is spleen venous sinus. For reference, the given name/label by SMEs is Splenic Sinusoid. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by anitacaron** [comment]
Spleen_1.4_DRAFT: https://docs.google.com/spreadsheets/d/1KugrRJwk_IlvOpCpDffvk9YKFbSFd4PtJgAzxgrAmtM/edit?pli=1#gid=69626346

1. **FIXED** The term CL:0002399 has a different name/label in the source ontology in the following 1 row 55. The name/label in the ASCT+B table is myeloid dendritic cell cDC2 and the one in the ontology is CD1c-positive myeloid dendritic cell. For reference, the given name/label by SMEs is cDC2 myeloid dendritic cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by emquardokus** [comment]
Spleen_1.4_DRAFT: https://docs.google.com/spreadsheets/d/1KugrRJwk_IlvOpCpDffvk9YKFbSFd4PtJgAzxgrAmtM/edit?pli=1#gid=69626346

**Miscellaneous fixes**
**References with "No DOI"--replace with citation ID of appropriate format**
O’Malley, Dennis P., Tracy I. George, Attilio Orazi, and Susan L. Abbondanzo. 2009. Benign and Reactive Conditions of Lymph Node and Spleen. American Registry of Pathology. https://doi.org/10.55418/9781933477060.	https://doi.org/10.55418/9781933477060.

Garrafa E, Trainini L, Benetti A, Saba E, Fezzardi L, Lorusso B, Borghetti P, Bottio T, Ceri E, Portolani N, Bonardlli S, Giulini SM, Annibale G, Corradi A, Imberti L, Caruso A. Isolation, purification, and heterogeneity of human lymphatic endothelial cells from different tissues. Lymphology. 2005 Dec;38(4):159-66. PMID: 16515224.	PMID:16515224

Guisado Vasco, Pablo, José L. Villar Rodríguez, José Ibañez Martínez, Ricardo González Cámpora, and Hugo Galera Davidson. 2009. “Immunohistochemical Organization Patterns of the Follicular Dendritic Cells, Myofibroblasts and Macrophages in the Human Spleen--New Considerations on the  Pathological Diagnosis of Splenectomy Pieces.” International Journal of Clinical and Experimental Pathology 3 (2): 189–202.	PMID: 20126587

Toccanier-Pelte MF, Skalli O, Kapanci Y, Gabbiani G. Characterization of stromal cells with myoid features in lymph nodes and spleen in normal and pathologic conditions. Am J Pathol. 1987 Oct;129(1):109-18. PMID: 3310649; PMCID: PMC1899700.	PMID: 3310649



**Different labels**
1. Removed space to fix 1 instance. The term CL:0000499 has a different name/label in the source ontology in the following 1 row 53. The name/label in the ASCT+B table is stromal cell _ and the one in the ontology is _stromal cell. For reference, the given name/label by SMEs is Stromal Cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

2. Fixed label CD1c-positive myeloid dendritic cell .The term CL:0002399 has a different name/label in the source ontology in the following 1 row 126. The name/label in the ASCT+B table is myeloid dendritic cell cDC2 and the one in the ontology is CD1c-positive myeloid dendritic cell. For reference, the given name/label by SMEs is cDC2 myeloid dendritic cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

3. Changed all 3 instances to Splenic Sinusoid	splenic sinusoid	UBERON:0003910 The term UBERON:0009967 has a different name/label in the source ontology in the following 2 rows 31, 32. The name/label in the ASCT+B table is splenic sinusoid and the one in the ontology is spleen venous sinus. For reference, the given name/label by SMEs is Splenic Sinusoid. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

4. Fixed CD141-positive myeloid dendritic cell The term CL:0002394 has a different name/label in the source ontology in the following 1 row 125. The name/label in the ASCT+B table is myeloid dendritic cell cDC1 and the one in the ontology is CD141-positive myeloid dendritic cell. For reference, the given name/label by SMEs is cDC1 myeloid dendritic cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Blank ontology ID**
1. ADDED label, ID to table. No term id was found for the name/label Catecholaminergic Neuron in the following 6 rows 13, 14, 98, 108, 134, 141.
2. ADDED ID to B cell	CL:0000236 No term id was found for the name/label B cell in the following 1 row 39.
3.ADDED Cytotoxic Memory T Cell	CD8-positive, alpha-beta memory T cell	CL:0000909 No term id was found for the name/label Cytotoxic Memory T Cell in the following 1 row 119.

**Blank ontology ID missing parent**
1. New term was added, catecholaminergic neuron	CL:4033050. The term Catecholaminergic Neuron without ontology ID has no parent that is from the CL ontology in the following 6 rows 13, 14, 98, 108, 134, 141.

2. Added parent stromal cell	CL:0000499; The term Adventitial Stromal Cell without ontology ID has no parent that is from the CL ontology in the following 2 rows 19, 97.

3. Added parent stromal cell	CL:0000499; The term Splenic Stromal Sheath Cell without ontology ID has no parent that is from the CL ontology in the following 1 row 25.

4. added parent stromal cell	stromal cell	CL:0000499 The term Reticular Cell of splenic sinusoid without ontology ID has no parent that is from the CL ontology in the following 1 row 30.

5. added parent vein endothelial cell	CL:0002543 The term Splenic Littoral Cell without ontology ID has no parent that is from the CL ontology in the following 1 row 31.

6. added parent stromal cell	stromal cell	 The term Splenic Red Pulp Stromal Cell without ontology ID has no parent that is from the CL ontology in the following 1 row 32.

7.  ADDED ID to B cell	CL:0000236 The term B cell without ontology ID has no parent that is from the CL ontology in the following 1 row 39.

8. added parent cell leukocyte	CL:0000738 The term Splenic perifollicular zone macrophage without ontology ID has no parent that is from the CL ontology in the following 1 row 52.

9. added parent conventional dendritic cell	CL:0000990.  The term DEC205+ DC without ontology ID has no parent that is from the CL ontology in the following 3 rows 58, 65, 127.

10. added parent B cell	B cell	CL:0000236 The term Mantle Zone B Cell without ontology ID has no parent that is from the CL ontology in the following 1 row 72.

11. Added parent mural cell	mural cell	CL:0008034aThe term Splenic Pericyte without ontology ID has no parent that is from the CL ontology in the following 2 rows 76, 80.

12.The term Interdigitating Dendritic Cells without ontology ID has no parent that is from the CL ontology in the following 1 row 106.

13. added parent smooth muscle cell	CL:0000192 The term Splenic Smooth Muscle Cell without ontology ID has no parent that is from the CL ontology in the following 1 row 110.

14. ADDED Cytotoxic Memory T Cell	CD8-positive, alpha-beta memory T cell	CL:0000909 The term Cytotoxic Memory T Cell without ontology ID has no parent that is from the CL ontology in the following 1 row 119.

15. The term Subcapsular Dendritic Cell without ontology ID has no parent that is from the CL ontology in the following 1 row 137.
